src/main/java/org/terasology/signalling/componentSystem/SignalSwitchBehaviourSystem.java

```diff
@@ -106,6 +106,10 @@
 
     private Map<String, GateSignalChangeHandler> signalChangeHandlers = Maps.newHashMap();
 
+    /**
+     * Prepares signalling related blocks and SignalChangeHandlers for later event handling.
+     * These are obtained from the BlockManager.class file and by overridding GateSignalChangeHandler, respectively.
```

Comment:
Protip: There are little formatting tools in Javadoc. For instance you could use a `@link` here for `BlockManager` and JavaDoc will turn it into an active hyperlink to the doc for that class. Neat!

See https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDDIECH for some details